### PR TITLE
exim: Add import and save stats

### PR DESCRIPTION
**Pull request by kingthorin**
Part of zaproxy/zaproxy#6579

Add stats for importing and saving files, individual URLs/messages, and errors.
Add output panel messages for starting and finishing files. (Can be handy for example when importing a file of URLs and encountering timeouts.)

Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

**Comment by thc202** [comment]
Does it really worth adding the output messages? Ideally it should be using the import panel.

**Comment by kingthorin** [comment]
You mean the progress thing?

I think it’s still handy to have the Output messages.

I’m just not sure when I or someone else might get to that work (it needs to be split off OpenAPI and added to commonlib, then implemented here).

**Comment by thc202** [comment]
OK.

File: addOns/exim/src/main/resources/org/zaproxy/addon/exim/resources/Messages.properties [hunk 1]
```diff
@@ -18,6 +21,9 @@ exim.importLogFiles.choosefile.message = Input type:
 exim.importLogFiles.choosefile.title = Select Input Type
 exim.importLogFiles.log.type.zap = ZAP Messages
 exim.importLogFiles.log.type.modsec2 = ModSecurity2 Logs
+exim.importLogFiles.output.start = Importing {0}
+exim.importLogFiles.output.end = Done importing {0}
+exim.importLogFiles.output.error = Error importing {0}
```
Comment (thc202) [inline addOns/exim/src/main/resources/org/zaproxy/addon/exim/resources/Messages.properties:26]: Could use generic keys instead and reuse for the two cases?

File: addOns/exim/src/main/java/org/zaproxy/addon/exim/log/LogsImporter.java [hunk 2]
```diff
@@ -226,20 +247,38 @@ private static void processZapLogs(List<String> parsedRequestAndResponse)
...
+                Stats.incCounter(ExtensionExim.STATS_PREFIX + STATS_ZAP_FILE_MSG_ERROR);
+            } else {
+                Stats.incCounter(ExtensionExim.STATS_PREFIX + STATS_MODSEC2_FILE_MSG_ERROR);
+            }
+        }
+    }
+
+    private static void updateOutput(String messageKey, String filePath) {
+        if (View.isInitialised()) {
+            StringBuilder sb = new StringBuilder();
```
Comment (thc202) [inline addOns/exim/src/main/java/org/zaproxy/addon/exim/log/LogsImporter.java:279]: Could initialise to some higher value than the default, the file path will surely be bigger than that. (Same comment for the other import.)

**Comment by kingthorin** [comment]
Addressed all those, thanks for the review.

**Review by psiinon — APPROVED** [review]

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**